semcore/accordion/src/index.d.ts

```diff
@@ -64,6 +64,13 @@ export type AccordionItemContext = {
 };
 
 export type AccordionItemToggleProps = BoxProps & KeyboardFocusProps;
+export type ChevronItemProps = BoxProps & {
+  /**
+   * Chevron size
+   * @ default m
```

Comment:
Please remove space between `@` and `default` to make this commend have effect for jsdoc